messages/jsonschema/GherkinDocument.json

```diff
@@ -347,8 +348,20 @@
           "description": "The location of the steps' `keyword`"
         },
         "keyword": {
+          "description": "The actual keyword as it appeared in the source.",
           "type": "string"
         },
+        "keywordType": {
+          "description": "The test phase signalled by the keyword: Contex definition (Given), Action performance (When), Outcome assertion (Then). Other keywords signal Continuation (And and But) from a prior keyword. Lastly, the `*` 'keyword` isn't associated with a specific phase; hence the association to 'General'.",
+          "type": "string",
+          "enum": [
+            "Context",
+            "Action",
+            "Outcome",
+            "Conjunction",
+            "General"
```

Comment:
I would prefer `Uknown` here to emphasise that we don't know the semantic keyword type.

Reply:
Here I think that `Unknown` is actually better than `Neutral` because it conveys that information isn't avaliable: there's no explicit mapping to exactly one of the enumerated types. So, if I get to point the bikeshed, it's `Unknown`.

Reply:
I think for `* ` there is a kind of consensus, that it should be neutral to the Context/Action/Outcome phases. The `Unknown` is hiding this fact, so from Aslak's choices my vote is for `Netural`, but I even better like `Any` suggested by @luke-hill.

Reply:
We can still have an additional `Unknown` as well, of course, but I would not use it for the `* `.

Reply:
@aslakhellesoy I'm continuing my work on this change. I've now picked up your preference to associate `* ` with `Unknown`. Let me know if that's not going to fly.